### PR TITLE
Post Model created

### PR DESCRIPTION
**Pull request by mhkmcp**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tanvirstreame) [inline .idea/misc.xml]: Remove these

File: .idea/modules.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tanvirstreame) [inline .idea/modules.xml]: Remove these

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tanvirstreame) [inline .idea/vcs.xml]: Remove these

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
[the remark this answers is not in the document]
Comment (mhkmcp) [reply .idea/inspectionProfiles/profiles_settings.xml]: this file is created automatically when i opened the project with vscode

[the remark this answers is not in the document]
Comment (tanvirstreame) [reply .idea/inspectionProfiles/profiles_settings.xml]: delete it

File: backend/core/views.py [hunk 5]
```diff
@@ -1,10 +1,18 @@
...
 class UserViewset(generics.ListCreateAPIView):
     serializer_class = UserSerializer
     queryset = CustomUser.objects.all()
 
+
+class PostViewset(generics.ListCreateAPIView):
+    serializer_class = PostSerializer
+    queryset = Post.objects.all()
+
+
```
Comment (tanvirstreame) [inline backend/core/views.py]: Remove extra blank line ..... keep 1 extra blank link after the code

**Pull request merged**